std/assembly/string.ts

```diff
@@ -11,6 +11,7 @@ import { idof } from "./builtins";
   @lazy static readonly MAX_LENGTH: i32 = BLOCK_MAXSIZE >>> alignof<u16>();
 
   static fromCharCode(unit: i32, surr: i32 = -1): string {
+    assert(unit >= 0);
```

Comment:
Seems JS simply converts this to unsigned:

```ts
> String.fromCharCode(-123123)
'Ἅ'
> String.fromCharCode(-123123 >>> 0)
'Ἅ'
```

Reply:
Yeah. Will improve this